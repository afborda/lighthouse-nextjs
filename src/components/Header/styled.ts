import styled from 'styled-components'

export const Container = styled.div`
   display: flex;
   justify-content: space-between;
   align-items: center;
   .pg-inicial {
      font-size: 1.6em;
      color: #9d9ea0;
   }
   .pg-title {
      font-size: 1.6em;
      color: #143c8d;
      font-weight: bold;
   }
   .botao-voltar {
      font-size: 1.6em;
      color: #143c8d;
      padding: 10px 15px 10px 15px;
      border: 1px solid #143c8d;
      border-radius: 30px;
      font-weight: bold;
   }
`
