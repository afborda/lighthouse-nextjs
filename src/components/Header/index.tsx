import React from 'react'

import { Container } from './styled'

const Header = () => {
   return (
      <Container>
         <div>
            <p className="pg-inicial">Pagina Inicial\</p>
            <h1 className="pg-title">Meds</h1>
         </div>
         <div className="container-botao">
            <a className="botao-voltar" href="#">
               Voltar
            </a>
         </div>
      </Container>
   )
}

export default Header
