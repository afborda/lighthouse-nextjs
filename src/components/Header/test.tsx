import { render, screen } from '@testing-library/react'
import Header from '.'

describe('<Main/>', () => {
   it('should render the heading', () => {
      const { container } = render(<Header />)

      expect(container.firstChild).toMatchSnapshot()
   })
})
