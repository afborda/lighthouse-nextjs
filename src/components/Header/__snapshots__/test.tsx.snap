// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main/> should render the heading 1`] = `
<div
  class="styled__Container-sc-1nq6g88-0 iGEFbm"
>
  <div>
    <p
      class="pg-inicial"
    >
      Pagina Inicial\\
    </p>
    <h1
      class="pg-title"
    >
      Meds
    </h1>
  </div>
  <div
    class="container-botao"
  >
    <a
      class="botao-voltar"
      href="#"
    >
      Voltar
    </a>
  </div>
</div>
`;
